astropy/io/votable/tree.py

```diff
@@ -3014,9 +3014,7 @@ def __init__(self, name=None, ID=None, utype=None, type='results',
 
     def __repr__(self):
         buff = io.StringIO()
-        XMLWriter(buff).element(
-            self._element_name,
-            attrib=w.object_attrs(self, self._attr_list))
+        Resource.to_xml(self, XMLWriter(buff))
```

Comment:
No particular reason, I was following the convention of the __repr__method of SimpleElement